lib/climbing-crag-forecast.js

```diff
@@ -0,0 +1,35 @@
+import { Forecast } from './forecast.js';
+import { Location } from './location.js';
+
+/**
+ * Represents a climbing crag forecast storage. 
+ */
+export default class ClimbingCragForecast {

```

Comment:
¿Cómo sabéis a qué zona se refiere esto?

Reply:
Debéis resolver los comentarios si no se cierran automáticamente.

Reply:
... y en este caso quizás podríais cambiar el nombre ya que se refiere a un país.

Reply:
Habéis resuelto este comentario. ¿Qué cambio habéis hecho para resolverlo?